appveyor.yml

```diff
@@ -0,0 +1,192 @@
+#
+# Copyright 2015 Benjamin Worpitz
+#
+# This file is part of alpaka.
+#
+# alpaka is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# alpaka is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with alpaka.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+################################################################################
+# General configuration.
+################################################################################
+
+version: "{build}"
+
+branches:
+    except:
+        - gh-pages
+
+# Build tags (GitHub only)
+skip_tags: false
+
+################################################################################
+# Environment configuration.
+################################################################################
+
+os:
+    - Visual Studio 2015
+
+init:
+    - cmd: cmake --version
+    - cmd: git --version
+    - cmd: msbuild /version
+    # Fixes the line endings on Windows.
+    - cmd: git config --global core.autocrlf true
+
+# Clone directory.
+clone_folder: c:\projects\alpaka
+# Fetch repository as zip archive.
+shallow_clone: false
+# Set the clone depth.
+clone_depth: 1
+
+environment:
+    global:
+        ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE: ON
+        ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE: ON
+        ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE: ON
+        ALPAKA_ACC_CPU_BT_OMP4_ENABLE: OFF
+        ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE: ON
+
+    matrix:
+        - ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF
+          ALPAKA_DEBUG: 2
```

Comment:
This spams a lot inside the check output. Is there a limit for the output file in `appveyor`?